### PR TITLE
Feature/card UI polish

### PR DESCRIPTION
**Pull request by brightiortsor**
[Netlify Demo Link](https://cheery-dasik-1bf627.netlify.app)

This PR adds the following:

1. Modified the Startup card UI styles
2. Modified the grid layout to properly fit all screen sizes
3. Added a function to truncate the bio card description text (This was done to maintain a uniform card height on all screens)
4. Added a NotFound UI for when search result is empty

**FUTURE MODIFICATION SUGGESTIONS**

- Add a startup details page where to display all details of that startup (i.e all that is left of it anyways).
- Pagination as against the current infinite scroll
- Integrate a database to be able automate the process of adding graveyard startups. This will also help to reduce human errors as someone might want to delibrately add falsified information. 
- Very importantly, we need to add a VERY LARGE DISCLAIMER at the footer `e get why`.

Screenshot (167)
Screenshot (168)
Screenshot (169)
Screenshot (170)

### CODE REVIEW

**Comment by brightiortsor** [comment]
@slightlynerd Hiii,
I have gone through your detailed justification for some of the initial modifications made and I agree with some actually.
Nonetheless, I have removed the bg color for both the card and not-found components as well as the truncated text func. These changes have been pushed accordingly. I hope we can bring on better ideas to improve the project going forward.

**Comment by slightlynerd** [comment]
@brightiortsor Sure, I'll take a look at the PR and give feedback.

File: src/components/startup-bio.vue [hunk 1]
```diff
@@ -50,8 +50,41 @@ export default defineComponent({
...
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.3);
+  transform: translateY(-5px);
+}
 .h4 {
   display: inline-block;
   border-bottom: 1px solid;
 }
+
+.shutdown {
+  background: #f0eded;
```
Comment (slightlynerd) [inline src/components/startup-bio.vue:72]: @brightiortsor Move the hex (#f0eded) to src/styles/assets/styles/variables.scss, import the file and then use it.

```scss
// in variables.scss
$bg-shutdown: #f0eded;

// in startup-bio.vue
<style lang="scss" scoped>
@import '@/assets/styles/variables.scss';

...

// usage
.shutdown {
  background: $bg-shutdown;
  ...
}
</style>
```

**Review by slightlynerd — CHANGES_REQUESTED** [review]

**Comment by slightlynerd** [comment]
@brightiortsor After making the changes, run `yarn lint` to fix lint errors.

**Comment by slightlynerd** [comment]
@brightiortsor For subsequent commits, please read the contribution guide and follow the format for commit messages.

**Pull request closed without merging**